app/models/project.rb

```diff
@@ -1,4 +1,17 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def self.avg_exp(project)
```

Comment:
Should this be a class method? When you have a class method in which you have to pass in an instance, that suggests it could be an instance method